Example/OCMockitoSwiftTests/OCMockitoSwiftTests.swift

```diff
@@ -1,6 +1,6 @@
 import Quick
 import Nimble
-import OCHamcrest
+//import OCHamcrest
```

Comment:
```suggestion
import OCHamcrest
```
Sorry that's my debugging leftover 😄